lib/Controller/Id4meController.php

```diff
@@ -46,6 +46,11 @@
 use OCP\IUserSession;
 use OCP\Security\ISecureRandom;
 
```

Comment:
Not a fan of this.

```
diff --git a/composer.json b/composer.json
index 64f65dc..017a998 100644
--- a/composer.json
+++ b/composer.json
@@ -45,7 +45,8 @@
         "classmap_directory": "/lib/autoload/",
         "classmap_prefix": "NEXTCLOUD_USER_OIDC_",
         "packages": [
-            "firebase/php-jwt"
+            "firebase/php-jwt",
+            "id4me/id4me-rp"
         ],
         "delete_vendor_directories": true
     }
```

should make them available through Mozart and then you should be able to use them with `OCA\\UserOIDC\\Vendor\\Id4me...`